__tests__/__unit__/dataset/actions.unit.test.ts

```diff
@@ -0,0 +1,1508 @@
+    Object.defineProperty(globals.LOG, "error", { value: jest.fn(), configurable: true });
+    Object.defineProperty(zowe, "Download", { value: jest.fn(), configurable: true });
+    Object.defineProperty(zowe.Download, "dataSet", { value: jest.fn(), configurable: true });
+    Object.defineProperty(zowe, "Delete", { value: jest.fn(), configurable: true });
+    Object.defineProperty(zowe.Delete, "dataSet", { value: jest.fn(), configurable: true });
+    Object.defineProperty(fs, "unlinkSync", { value: jest.fn(), configurable: true });
+    Object.defineProperty(fs, "existsSync", { value: jest.fn(), configurable: true });
+    Object.defineProperty(sharedUtils, "concatChildNodes", { value: jest.fn(), configurable: true });
+    Object.defineProperty(Profiles, "getInstance", { value: jest.fn(), configurable: true });
+    Object.defineProperty(zowe, "List", { value: jest.fn(), configurable: true });
+    Object.defineProperty(zowe.List, "dataSet", { value: jest.fn(), configurable: true });
+    Object.defineProperty(vscode, "ProgressLocation", { value: jest.fn(), configurable: true });
+    Object.defineProperty(vscode.window, "createWebviewPanel", { value: jest.fn(), configurable: true });
+    Object.defineProperty(vscode.env, "clipboard", { value: clipboard, configurable: true });
+}
+
+// Idea is borrowed from: https://github.com/kulshekhar/ts-jest/blob/master/src/util/testing.ts
+const mocked = <T extends (...args: any[]) => any>(fn: T): jest.Mock<ReturnType<T>> => fn as any;
+
+describe("Dataset Actions Unit Tests - Function createMember", () => {
+    let blockMocks;
+
+    function createBlockMocks() {
+        const session = createISession();
+        const imperativeProfile = createIProfile();
+        const zosmfSession = createBasicZosmfSession(imperativeProfile);
+        const treeView = createTreeView();
+        const datasetSessionNode = createDatasetSessionNode(session, imperativeProfile);
+        const mvsApi = createMvsApi(imperativeProfile);
+        bindMvsApi(mvsApi);
+
+        return {
+            session,
+            zosmfSession,
+            treeView,
+            imperativeProfile,
+            datasetSessionNode,
+            mvsApi,
+            testDatasetTree: createDatasetTree(datasetSessionNode, treeView)
+        };
+    }
+
+    afterAll(() => jest.restoreAllMocks());
+
+    it("Checking of common dataset member creation", async () => {
+        createGlobalMocks();
+        blockMocks = createBlockMocks();
+        const parent = new ZoweDatasetNode("parent", vscode.TreeItemCollapsibleState.Collapsed,
+            blockMocks.datasetSessionNode, blockMocks.session);
+
+        mocked(vscode.window.showInputBox).mockResolvedValue("testMember");
+        mocked(vscode.window.withProgress).mockImplementation((progLocation, callback) => {
+            return callback();
+        });
+        jest.spyOn(blockMocks.mvsApi, "getContents").mockResolvedValueOnce({
```

Comment:
I believe we can get rid of spyOn and directly override the mock, because we always create the mocks from scratch. Do you agree @stepanzharychevbroadcom ?

Reply:
@VitGottwald It's how it was used in all tests and I used the same style as a result. To get rid of that I will need to go through all of the tests.

Reply:
OK, then let's leave it for the next round of changes. I saw that @katelynienaber did that in her tests already, but I do not want to cause too much churn.